include/aspect/geometry_model/interface.h

```diff
@@ -26,12 +26,20 @@
 #include <deal.II/base/parameter_handler.h>
 #include <deal.II/distributed/tria.h>
 #include <deal.II/base/std_cxx11/array.h>
+#include <aspect/utilities.h>
 
 #include <set>
 
 
 namespace aspect
 {
+  namespace Utilities
+  {
+    namespace Coordinates
+    {
+      enum CoordinateSystem : unsigned short;
```

Comment:
Do you need this in addition to `#include`ing `aspect/utilities.h`? If not, remove the duplicate declaration.

Reply:
I can't remove it somehow, even though it is defined in `aspect/utilities.h` which is included here at the top.

Reply:
If you remove it, what is the error message?